packages/SwingSet/src/kernel/kernel.js

```diff
@@ -574,12 +658,25 @@ export default function buildKernel(kernelEndowments) {
       });
     }
 
+    // evaluate the outer vatAdmin vat for two registration functions
+    const { endowments: endow, registerVatCreationFunction } = buildVatAdmin();
+
+    if (vatAdminDevSetup) {
```

Comment:
same as above, let's move this into a clause that adds the device to `genesisDevices`